waspc/src/Wasp/Generator/FileDraft.hs

```diff
@@ -24,12 +26,14 @@ import Wasp.Generator.Templates (TemplatesDir)
 data FileDraft
```

Comment:
Looking at this, it would make sense to have `FileDraft` and `DirDraft` instead of putting dir draft together with file drafts. Unless we e.g. renamed `FileDraft` to `FileSystemItemDraft`? Or we can just agree that `FileDraft` actually implicitly refers to the file system element instead of the specific file.

Reply:
Yeah, this is on spot.
The thing is, refactoring here sounds like a PITA, so maybe just putting a nice comment on FileDraft would help enough for now. Although it is a bit of "hey we are in debt here so let's take another loan to cover it" :D.

Reply:
Yeah, agree it somewhat muddles the name. In true "everything is a file" Linux fashion, directories are technically still files if you squint enough :D But agree, better to comment and be explicit to avoid confusion. And agree that is a pretty big refactoring effort to do now.